### PR TITLE
feat/unsafe html

### PR DESCRIPTION
**Pull request by egor-romanov**
# Changes

- Add ability to send emails with unsafe html in markdown bodies.
- Also enables some extensions from goldmark.
- Hidden behind env, flag `UNSAFE_HTML` (`--unsafe`, `-u`).

Enable yo to send emails with a bit of styling to set width for images for example. You can add styles to your signature.

```markdown
---

Best, Egor R.

<style>
img{
    width:100%;
}
p,a,ul,h3{
    font-size:16px;
}
table,pre{
    background-color:#181818;
    color:#cccccc;
    font-size:14px;
    padding:16px;
}
</style>
```

### CODE REVIEW

**Comment by egor-romanov** [comment]
- before
before

- after
after

File: main.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ import (
 )
 
 const RESEND_API_KEY = "RESEND_API_KEY"
+const UNSAFE_HTML = "UNSAFE_HTML"
```
Comment (maaslalani) [inline main.go:18]:
```suggestion
const UNSAFE_HTML = "POP_UNSAFE_HTML"
```

**Comment by maaslalani** [comment]
This is fantastic! Thanks so much for this PR.

**Comment by egor-romanov** [comment]
thx!

**Pull request merged**